[PATCH 5/6] Fix "seems to be moved"
* libltdl/config/ltmain.m4sh (func_mode_link): Compare files by inode
to fix "seems to be moved" warning.
---
 libltdl/config/ltmain.m4sh |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/libltdl/config/ltmain.m4sh b/libltdl/config/ltmain.m4sh
index af46cb8..244bb5b 100644
--- a/libltdl/config/ltmain.m4sh
+++ b/libltdl/config/ltmain.m4sh
@@ -6283,7 +6283,9 @@ func_mode_link ()
 		  eval libdir=`${SED} -n -e 's/^libdir=\(.*\)$/\1/p' $deplib`
 		  test -z "$libdir" && \
 		    func_fatal_error "\`$deplib' is not a valid libtool archive"
-		  test "$absdir" != "$libdir" && \
+		  abs_inode=`ls -i "$deplib" | awk '{print $1}'`
+		    lib_inode=`ls -i "$libdir/$(basename $deplib)" | awk '{print $1}'`
+		  test "$abs_inode" != "$lib_inode" && \
 		    func_warning "\`$deplib' seems to be moved"
 
 		  path="-L$absdir"
-- 
1.7.0.2.msysgit.0